friends/friendModel.js

```diff
@@ -0,0 +1,39 @@
+//here we'll create a shape for all our documents that come in. We will build a model for bears. 
+const mongoose = require('mongoose');
+
+//The model is where we define the schema. It's a blueprint of what our collection will look like
+// collection will be what our model is called
+
+// Schema
+// {
+//     firstName: "Jane", // String, required
+//     lastName: "Doe",  // String, required
+//     age: 18, // Number, required, should be an integer between 1 and 120
+//     createdOn: Mon Aug 14 2017 12:50:16 GMT-0700 (PDT) // Date, required, defaults to current date
+//   }
+
+const FriendSchema = new mongoose.Schema({
+    firstName: {
+        type: String,
+        required: true
```

Comment:
I like to leave a comma after the last property in each of these objects -- helps to avoid errors when I come in and add another property. NOT important, just a style tip that you can take or leave 😄